eya_def_tools/eya_def_tools/tests/conftest.py

```diff
@@ -1422,53 +1451,74 @@ def energy_assessment_a(
+                                    statistic_type=(
+                                        dataset.StatisticType.STANDARD_DEVIATION
+                                    ),
+                                    return_period=10.0,
                                 ),
                                 values=3.4681,
                             ),
                             dataset.DatasetStatistic(
-                                statistic_type=dataset.ExceedanceLevelStatisticType(
-                                    exceedance_level=0.9
+                                statistic=dataset.SimpleStatistic(
+                                    statistic_type=(
+                                        dataset.StatisticType.STANDARD_DEVIATION
+                                    ),
+                                    return_period=1.0,
                                 ),
-                                values=27.0894,
-                            ),
-                        ],
-                    ),
-                    dataset.Dataset(
-                        assessment_period=dataset.AssessmentPeriod.ANY_ONE_YEAR,
-                        dimensions=None,
-                        statistics=[
-                            dataset.DatasetStatistic(
-                                statistic_type=dataset.BasicStatisticType.MEDIAN,
-                                values=31.5286,
+                                values=4.7293,
                             ),
                             dataset.DatasetStatistic(
-                                statistic_type=(
-                                    dataset.BasicStatisticType.STANDARD_DEVIATION
+                                statistic=dataset.ExceedanceLevelStatistic(
+                                    statistic_type=(
+                                        dataset.StatisticType.EXCEEDANCE_LEVEL
+                                    ),
+                                    probability=0.9,
+                                    return_period=10.0,
                                 ),
-                                values=4.7293,
+                                values=27.0894,
                             ),
                             dataset.DatasetStatistic(
-                                statistic_type=dataset.ExceedanceLevelStatisticType(
-                                    exceedance_level=0.9
+                                statistic=dataset.ExceedanceLevelStatistic(
+                                    statistic_type=(
+                                        dataset.StatisticType.EXCEEDANCE_LEVEL
+                                    ),
+                                    probability=0.9,
+                                    return_period=1.0,
                                 ),
                                 values=25.4751,
                             ),
                         ],
                     ),
                 ],
+                energy_production=[
```

Comment:
This is just an initial "dummy" result with a single value. We can work separately on building more complete examples.